pkgs/applications/networking/calls/default.nix

```diff
@@ -62,11 +64,17 @@ stdenv.mkDerivation rec {
     folks
     gom
     gsound
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
     feedbackd
     callaudiod
     gtk3
     libpeas
     libgdata # required by some dependency transitively
```

Comment:
Which one?

Reply:
I don't know. Should that be kept separate from the 41.1 upgrade PR?

Reply:
It would be nice to know which other packages makes this a transitively dependency so that we can propagate it there. If we know the package we can open another PR which cleans up similar workarounds.

Reply:
WIP: https://github.com/NixOS/nixpkgs/pull/150044

Reply:
https://github.com/NixOS/nixpkgs/pull/150044/files will fix this for `pkgs.calls` and other packages impacted by this.